crates/kubelet/src/handle.rs

```diff
@@ -93,17 +94,22 @@ pub struct PodHandle<S, R> {
     container_handles: RwLock<HashMap<String, RuntimeHandle<S, R>>>,
     status_handle: JoinHandle<()>,
     pod: Pod,
+    // Storage for the volume references so they don't get dropped until the runtime handle is
+    // dropped
+    _volumes: Option<HashMap<String, VolumeRef>>,
```

Comment:
Is there a logical difference between `volumes == None` and `volumes == Some(HashMap::default())`?

Reply:
That is a good point. I should just unwrap_or_default the option that is passed in